packages/relay/src/lib/eth.ts

```diff
@@ -1567,16 +1566,34 @@ export class EthImpl implements Eth {
     let fileId: FileId | null = null;
     let txSubmitted = false;
     try {
-      const sendRawTransactionResult = await this.hapiService
-        .getSDKClient()
-        .submitEthereumTransaction(
-          transactionBuffer,
-          EthImpl.ethSendRawTransaction,
-          requestDetails,
-          originalCallerAddress,
-          networkGasPriceInWeiBars,
-          await this.getCurrentNetworkExchangeRateInCents(requestDetails),
-        );
+      const sendRawTransactionResult = await this.sendRawTransactionWithRetry(
```

Comment:
We should not be retrying `eth_sendRawTransaction`.
We should leave users and clients to do that based on the initial attempt response.
Retrying can add unintended latency and cost implications.
The transaction either fails gracefully or succeeds.
Let's discuss.

Reply:
@Nana-EC we only retry if we get a timeout exceeded and the transaction is not found in the mirror node
On [this](https://github.com/hashgraph/hedera-json-rpc-relay/pull/3161/files#diff-dd8c7990f7764960d05dcb4c1b5ae7eee48b425b1b83d6dcc7e7ccc442330ef0R1582) line you can check on what occasions we retry. Does this make sense?

Reply:
It does however that exact scenario highlights the scenarios I'm referring to.
For instance if the try logic now makes the E2E time over the timeout set by a cloud provider or the users client the experience wouldn't be great and it would spend unnecessary hbars and lead to the current confusion of a successful transaction on network but with a failure response.

Reply:
I think you should pull the retry logic out.
Check for a timeout occurrence and check if the MN has the transactions within a reasonable period.
If the transaction is found the transaction was successful and we shouldn't fail, if it's not found then throw the original timeout error back.